src/install_update.ts

```diff
@@ -61,24 +66,28 @@ export async function createUpdateResp(
 
   try {
     const cwd = Deno.cwd();
+    const dependencyFiles = await gatherDependencyFiles(cwd);
+    let updateResponses: Update[] = [];
 
-    // Update import_map.json with latest dependency versions
-    const importsUpdateResp = await updateDependencyFile(
-      `${cwd}/import_map.json`,
-      releases,
-    );
-
-    // Update slack.json with latest dependency versions
-    const hooksUpdateResp = await updateDependencyFile(
-      `${cwd}/slack.json`,
-      releases,
-    );
-
-    updateResp.updates = [...importsUpdateResp, ...hooksUpdateResp];
+    for (const [file, _] of dependencyFiles) {
+      // Update dependency file with latest dependency versions
+      const updateResp = await updateDependencyFile(`${cwd}/${file}`, releases);
+      updateResponses = [...updateResponses, ...updateResp];
+    }
+    updateResp.updates = updateResponses;
   } catch (err) {
     updateResp.error = { message: err.message };
   }
 
+  // Reduce the total updates returned by removing duplicates
+  updateResp.updates = updateResp.updates.reduce(
+    (existingUpdates: Update[], newUpdate) =>
+      existingUpdates.find((eu) => eu.name === newUpdate.name)
+        ? existingUpdates
+        : [...existingUpdates, newUpdate],
+    [],
+  );
```

Comment:
total nit: We could leverage a Set here to ensure uniqueness since this array isn't too big
```ts
updateResp.updates = [...new Set(updateResp.updates)];
```

Personal preference since I think it's a neat trick and `reduce` seems to trip some people (like me) up

Reply:
Oh, great suggestion! Much better than using `reduce` and significantly less verbose and more readable.